asn1acn.cpp

```diff
@@ -109,6 +119,29 @@ ExtensionSystem::IPlugin::ShutdownFlag Asn1AcnPlugin::aboutToShutdown()
     return SynchronousShutdown;
 }
 
+void Asn1AcnPlugin::loadSettings()
```

Comment:
it does not load but sets defaults, so maybe it should be named like: setDefaultSettings or loadDefaultSettings or somehting better :)

I also wonder - maybe it should be more "local"? In serviceprovider?